sql/catalyst/src/main/java/org/apache/spark/sql/connector/read/SupportsPushDownAggregates.java

```diff
@@ -51,6 +49,8 @@
    * Pushes down Aggregation to datasource. The order of the datasource scan output columns should
    * be: grouping columns, aggregate columns (in the same order as the aggregate functions in
    * the given Aggregation).
+   *
+   * Returns true if the aggregation can be pushed down to datasource.
```

Comment:
nit: maybe "@return true if the aggregation can be pushed down to datasource. False otherwise."

Reply:
Done. Thanks!